i18n/en/docusaurus-plugin-content-docs/current/guides/how-to-add-testing-library.mdx

```diff
@@ -0,0 +1,65 @@
+# How to Connect testing-library in testplane
+
+## Introduction {#preface}
+
+[Testing-library][testing-library] is a collection of tools for testing web application user interfaces, focused on creating reliable and maintainable tests by emphasizing user behavior. The main advantage of `testing-library` is its focus on interaction with interface elements. And in testplane, you can use the element search methods provided by the `testing-library` itself.
+
+## Connection
+
+To be able to use [queries][queries] from `testing-library` in testplane tests, you only need to follow a few steps.
+
+1. Install the npm package `'@testing-library/webdriverio'`
+
+```bash
+npm i -D @testing-library/webdriverio
+```
+
+2. Connect the implementation in the testplane configuration in the `prepareBrowser` section
+
+```javascript
+// .testplane.conf.js
+const { setupBrowser } = require("@testing-library/webdriverio");
+
+module.exports = {
+    prepareBrowser(browser) {
+        setupBrowser(browser);
+    },
+
+    // other testplane settings...
+};
+```
+
+## Usage
+
+After connecting, you will be able to use the search by selectors from `testing-library`, as described in the [official documentation](https://testing-library.com/docs/queries/about/). For example, searching for an element by its text:
```

Comment:
"After connecting". "connecting" should be translated as "configuration", "setup" or "installation"